articles/machines/qc-trace-simulator/depth-counter.md

```diff
@@ -27,11 +27,10 @@ of the <xref:microsoft.quantum.primitive.ccnot> operation. We will use the follo
 ```qsharp
 open Microsoft.Quantum.Primitive;
 operation CCNOTDriver() : Unit {
-    body (...) {
-        using (qubits = Qubit[3]) {
-            CCNOT(qubits[0], qubits[1], qubits[2]);
-            T(qubits[0]);
-        }
+
```

Comment:
Are we keeping the blank lines?

Reply:
I tried to be consistent with what the formatter does.